src/utils/cookie.js

```diff
@@ -26,4 +26,28 @@ function clearCookie(name) {
     createCookie(name, '', -1)
 }
 
-export { createCookie, readCookie, clearCookie }
+// lấy ra cookie với tên được truyền vào
+function getCookie(cname) {
+    let name = cname + '='
+    let decodedCookie = decodeURIComponent(document.cookie)
+    let ca = decodedCookie.split(';')
+    for (let i = 0; i < ca.length; i++) {
+        let c = ca[i]
+        while (c.charAt(0) == ' ') {
+            c = c.substring(1)
+        }
+        if (c.indexOf(name) == 0) {
+            return c.substring(name.length, c.length)
+        }
+    }
+    return ''
+}
+
+// condition cho việc check cookie
+function checkCookie(nameAccessC, nameRefreshC) {
+    let accessC = getCookie(nameAccessC)
+    let refreshC = getCookie(nameRefreshC)
+    return accessC != '' && refreshC != ''
```

Comment:
We don't need this function. You can use state `isLogin` from redux to check instead of this one.

```js
const { isLogin } = useShallowEqualSelector((state) => state.auth)
```

If state `isLogin` is true then the cookie exists both `accessToken` and `refreshToken`, otherwise not.